<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.1.251">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>Josiah’s Remote Sensing Learning Diary - 3&nbsp; Week 3: Corrections</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1.6em;
  vertical-align: middle;
}
</style>


<script src="site_libs/quarto-nav/quarto-nav.js"></script>
<script src="site_libs/quarto-nav/headroom.min.js"></script>
<script src="site_libs/clipboard/clipboard.min.js"></script>
<script src="site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="site_libs/quarto-search/fuse.min.js"></script>
<script src="site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="./">
<link href="./references.html" rel="next">
<link href="./week2.html" rel="prev">
<script src="site_libs/quarto-html/quarto.js"></script>
<script src="site_libs/quarto-html/popper.min.js"></script>
<script src="site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="site_libs/quarto-html/anchor.min.js"></script>
<link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="site_libs/bootstrap/bootstrap.min.js"></script>
<link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "sidebar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "start",
  "type": "textbox",
  "limit": 20,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit"
  }
}</script>

  <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml-full.js" type="text/javascript"></script>

</head>

<body class="nav-sidebar floating">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
  <nav class="quarto-secondary-nav" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
    <div class="container-fluid d-flex justify-content-between">
      <h1 class="quarto-secondary-nav-title"><span class="chapter-number">3</span>&nbsp; <span class="chapter-title">Week 3: Corrections</span></h1>
      <button type="button" class="quarto-btn-toggle btn" aria-label="Show secondary navigation">
        <i class="bi bi-chevron-right"></i>
      </button>
    </div>
  </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article">
<!-- sidebar -->
  <nav id="quarto-sidebar" class="sidebar collapse sidebar-navigation floating overflow-auto">
    <div class="pt-lg-2 mt-2 text-left sidebar-header">
    <div class="sidebar-title mb-0 py-0">
      <a href="./">Josiah’s Remote Sensing Learning Diary</a> 
    </div>
      </div>
      <div class="mt-2 flex-shrink-0 align-items-center">
        <div class="sidebar-search">
        <div id="quarto-search" class="" title="Search"></div>
        </div>
      </div>
    <div class="sidebar-menu-container"> 
    <ul class="list-unstyled mt-1">
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./index.html" class="sidebar-item-text sidebar-link">Preface</a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./week1.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">1</span>&nbsp; <span class="chapter-title">Week 1: Introduction to Remote Sensing</span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./week2.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">2</span>&nbsp; <span class="chapter-title">Week 2: Quarto and Xaringan</span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./week3.html" class="sidebar-item-text sidebar-link active"><span class="chapter-number">3</span>&nbsp; <span class="chapter-title">Week 3: Corrections</span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./references.html" class="sidebar-item-text sidebar-link">References</a>
  </div>
</li>
    </ul>
    </div>
</nav>
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">Table of contents</h2>
   
  <ul>
  <li><a href="#learning-objectives" id="toc-learning-objectives" class="nav-link active" data-scroll-target="#learning-objectives"><span class="toc-section-number">3.1</span>  Learning Objectives</a></li>
  <li><a href="#summary-of-key-concepts" id="toc-summary-of-key-concepts" class="nav-link" data-scroll-target="#summary-of-key-concepts"><span class="toc-section-number">3.2</span>  Summary of Key Concepts</a>
  <ul class="collapse">
  <li><a href="#overview" id="toc-overview" class="nav-link" data-scroll-target="#overview"><span class="toc-section-number">3.2.1</span>  Overview</a></li>
  <li><a href="#geometric-corrections-and-mosaicking-jensen-2015" id="toc-geometric-corrections-and-mosaicking-jensen-2015" class="nav-link" data-scroll-target="#geometric-corrections-and-mosaicking-jensen-2015"><span class="toc-section-number">3.2.2</span>  Geometric Corrections and Mosaicking (Jensen 2015)</a></li>
  </ul></li>
  <li><a href="#summary-of-practical" id="toc-summary-of-practical" class="nav-link" data-scroll-target="#summary-of-practical"><span class="toc-section-number">3.3</span>  Summary of Practical</a>
  <ul class="collapse">
  <li><a href="#mosaicking" id="toc-mosaicking" class="nav-link" data-scroll-target="#mosaicking"><span class="toc-section-number">3.3.1</span>  Mosaicking</a></li>
  <li><a href="#ratio-enhancements-ndvi" id="toc-ratio-enhancements-ndvi" class="nav-link" data-scroll-target="#ratio-enhancements-ndvi"><span class="toc-section-number">3.3.2</span>  Ratio enhancements: NDVI</a></li>
  </ul></li>
  <li><a href="#applications-of-ratio-enhancements" id="toc-applications-of-ratio-enhancements" class="nav-link" data-scroll-target="#applications-of-ratio-enhancements"><span class="toc-section-number">3.4</span>  Applications of Ratio Enhancements</a></li>
  <li><a href="#reflections" id="toc-reflections" class="nav-link" data-scroll-target="#reflections"><span class="toc-section-number">3.5</span>  Reflections</a></li>
  <li><a href="#references" id="toc-references" class="nav-link" data-scroll-target="#references"><span class="toc-section-number">3.6</span>  References</a></li>
  </ul>
</nav>
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title d-none d-lg-block"><span class="chapter-number">3</span>&nbsp; <span class="chapter-title">Week 3: Corrections</span></h1>
</div>



<div class="quarto-title-meta">

    
    
  </div>
  

</header>

<section id="learning-objectives" class="level2" data-number="3.1">
<h2 data-number="3.1" class="anchored" data-anchor-id="learning-objectives"><span class="header-section-number">3.1</span> Learning Objectives</h2>
<ul>
<li><p>Understand and apply types of corrections that can be applied to remotely sensed data</p></li>
<li><p>Understand and apply methods of data joining and enhancement</p></li>
<li><p>Understand how enhancements can be used to emphasize certain features or spectral traits</p></li>
</ul>
</section>
<section id="summary-of-key-concepts" class="level2" data-number="3.2">
<h2 data-number="3.2" class="anchored" data-anchor-id="summary-of-key-concepts"><span class="header-section-number">3.2</span> Summary of Key Concepts</h2>
<section id="overview" class="level3" data-number="3.2.1">
<h3 data-number="3.2.1" class="anchored" data-anchor-id="overview"><span class="header-section-number">3.2.1</span> Overview</h3>
<p>In this lecture, we mainly looked at types of corrections that could be applied to remotely sensed data:</p>
<ol type="1">
<li>Geometric
<ul>
<li><p>Use Ground Control Points (GCPs) to match points in the new image to a reference dataset.</p></li>
<li><p>Ideally done with backward mapping (output to input) so that we can get a value in the original input image for every value in the gold standard (Jensen 2015:247).</p></li>
<li><p>And resample the final raster</p></li>
</ul></li>
<li>Atmospheric
<ul>
<li><p>Relative (e.g.&nbsp;Dark Object Subtraction or Pseudo-Invariant Features)</p></li>
<li><p>Absolute; using atmospheric radiative transfer models, assuming atmospheric measurements are available (e.g.&nbsp;Py6S)</p></li>
</ul></li>
<li>Orthorectification/ Topographic Correction
<ul>
<li><p>Removing distortions such that the pixels viewed at nadir (straight down)</p></li>
<li><p>Requires sensor geometry and DEM.</p></li>
<li><p>Cosine correction can be used (Jensen 2015)</p></li>
</ul></li>
<li>Radiometric
<ul>
<li>Using Digital Number to obtain spectral radiance</li>
</ul></li>
</ol>
<p>We also looked at how remotely sensed data can be <strong>mosaicked</strong> and <strong>enhanced</strong>. Mosaicking is the process of joining 2 or more images together, similar to merging in the sense of polygons.</p>
<p>Types of Enhancements and methods:</p>
<ul>
<li><p>Contrast enhancement:</p>
<ul>
<li><p>Minimum-Maximum</p></li>
<li><p>Percentage Linear and Standard Deviation</p></li>
<li><p>Piecewise Linear Contrast Stretch</p></li>
</ul></li>
<li><p>Band Ratioing</p></li>
<li><p>Filtering</p></li>
<li><p>Principal Component Analysis (PCA)</p></li>
</ul>
</section>
<section id="geometric-corrections-and-mosaicking-jensen-2015" class="level3" data-number="3.2.2">
<h3 data-number="3.2.2" class="anchored" data-anchor-id="geometric-corrections-and-mosaicking-jensen-2015"><span class="header-section-number">3.2.2</span> Geometric Corrections and Mosaicking (Jensen 2015)</h3>
<p>For this week’s learning diary, I will be focusing on types of geometric corrections and the mosaicking process, based on Jensen’s textbook (2015).</p>
<p><strong>Ground Control Points</strong> are locations on the road surface that can be easily and accurately identified on a map. Each GCP should have 2 distinct sets of coordinates, image and map coordinates.</p>
<p><strong>Image-to-map rectification</strong></p>
<p>This is the process by which the geometry of an image is made planimetric, to remove distortion caused by topographic relief displacement.</p>
<p>Steps:</p>
<ol type="1">
<li><em>Spatial Interpolation:</em> The geometric relationship between input pixel coordinates and reference map coordinates must be identified, to establish the nature of transformation to be applied to all other pixels.
<ul>
<li><p>Unsystematic errors in the new image produced by changes in attitude (roll, pitch and yaw) or altitude</p></li>
<li><p>A first-order, six-parameter, linear transformation is sufficient to rectify the imagery to a geographic frame of reference.</p></li>
<li><p>This type of transformation can model 6 kinds of distortions (Novak 1992; Buiten and Van Putten 1997):</p>
<ul>
<li><p>Translation in x and y</p></li>
<li><p>Scale changes in x and y</p></li>
<li><p>Skew</p></li>
<li><p>Rotation</p></li>
</ul></li>
<li><p>Forward Mapping (input-to-output):</p>
<ul>
<li><p><span class="math display">\[
x = a_0 + a_1 x' + a_2 y'
\]</span></p></li>
<li><p><span class="math display">\[
y = b_0 + b_1 x' + b_2 y'
\]</span></p></li>
<li><p>This forward mapping logic is useful when we rectify the location of discrete coordinates along a linear feature. However, when we are filling a rectified output grid with values from an unrectified input image, forward mapping logic is not very useful as the output location may not fall exactly on a x,y output map coordinate.</p></li>
<li><p>This can result in output matrix pixels with no output values (Wolberg 1990)</p></li>
</ul></li>
<li><p>Inverse Mapping (output-to-input)</p>
<ul>
<li><p><span class="math display">\[
x' = a_0 + a_1 x + a_2 y
\]</span></p></li>
<li><p><span class="math display">\[
y' = b_0 + b_1 x + b_2 y
\]</span></p></li>
<li><p>The rectified output matrix is filled systematically, with the equation using the six coefficient to determine where to get a value from the original input image. Here, nearest-neighbour resampling logic is used.</p></li>
<li><p>A quadratic polynomial can also be used for transformations, but this is usually only used when there are serious geometric errors, usually in imagery obtained from suborbital aerial platforms.</p></li>
<li><p><span class="math display">\[
x' = c_0 + c_1x + c_2y + c_3xy + c_4x^2 + c_5y^2
\]</span></p></li>
<li><p><span class="math display">\[
y' = d_0 + d_1x + d_2y + d_3xy + d_4x^2 + d_5y^2
\]</span></p></li>
<li><p><strong>Computing the Root-Means-Squared-Error of the Inverse Mapping Function</strong></p></li>
<li><p>Using RMSE allows us to determine how well the 6 coefficients derived from the least-squares regression of the initial GCPs account for the geometric distortion in the input image.</p></li>
<li><p>The user specifies a threshold of acceptable total RSME, and the GCP with the most individual error will be deleted, then recompute the 6 coefficients and RMSE for all points, until the RMSE is less than the threshold or there are too few GCPs remaining for a regression.</p></li>
</ul></li>
</ul></li>
<li><em>Intensity Interpolation</em>: Pixel brightness values should be determined. When a pixel in the rectified output image requires a value from the input pixel that does not fall neatly on a row-and-column coordinate, there must be a mechanism for determining the Brightness Value (BV) to be assigned to the output rectified pixel.
<ul>
<li><p>Several methods of Brightness Value interpolation including <em>nearest neighbour, bilinear interpolation and cubic convolution</em></p></li>
<li><p>Nearest neighbour is computationally efficient and does not alter BVs during resampling. It should be used when biophysical information is to be extracted from the dataset.</p></li>
<li><p>Bilinear interpolation assigns pixel values by interpolating BVs in 2 orthogonal directions, computing a new BV based on weighted distances to the nearest 4 pixel values. This method acts as a spatial moving filter that subdues extremes in BVs in the output image.</p></li>
<li><p>Cubic convolution assigns values similarly to bilinear interpolation, except that weighted values of 16 input pixels are used.</p></li>
</ul></li>
</ol>
<p><strong>Image-to-image registration</strong></p>
<p>This is the translation and rotation alignment process by which 2 images of similar geometry and of same geographic area are positioned coincident to each other so that corresponding elements appear in the same place. This is used when it is not necessary to have each pixel assigned a unique x,y coordinate. A hybrid approach that uses both image-to-map rectification and image-to-image registration might be useful when detecting change between 2 or more dates of remotely sensed data.</p>
</section>
</section>
<section id="summary-of-practical" class="level2" data-number="3.3">
<h2 data-number="3.3" class="anchored" data-anchor-id="summary-of-practical"><span class="header-section-number">3.3</span> Summary of Practical</h2>
<p>This practical covered basic correction concepts such as atmospheric correction, mosaicking and enhancements. For this practical, I decided to look at Landsat8 imagery of Serengeti National Park (SNP) in Tanzania. I will be summarising my takeaways from the mosaicking and ratio enhancement sections of the practical, as I found them to be particularly useful. I was also unable to perform the PCA section as it took too long for my computer.</p>
<section id="mosaicking" class="level3" data-number="3.3.1">
<h3 data-number="3.3.1" class="anchored" data-anchor-id="mosaicking"><span class="header-section-number">3.3.1</span> Mosaicking</h3>
<p>For SNP, 3 tiles were needed to have a fuller view of the whole national park, so I downloaded 3 tiles that covered most of it. The first image shows the extent covered by the downloaded tiles, while the second shows the mosaicked output from R.</p>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="img/extent.png" class="img-fluid figure-img" style="width:100.0%"></p>
<p></p><figcaption class="figure-caption">Fig. 1: Extent of tiles</figcaption><p></p>
</figure>
</div>
</div>
</div>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="img/mosaic%20exten.png" class="img-fluid figure-img" style="width:100.0%"></p>
<p></p><figcaption class="figure-caption">Fig. 2: Mosaicked output</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>The mosaicked output in R was written to a GeoTiff file and then viewed in QGIS. It appears slightly off but I am unsure what went wrong in the process or this is an expected outcome.</p>
</section>
<section id="ratio-enhancements-ndvi" class="level3" data-number="3.3.2">
<h3 data-number="3.3.2" class="anchored" data-anchor-id="ratio-enhancements-ndvi"><span class="header-section-number">3.3.2</span> Ratio enhancements: NDVI</h3>
<p>The <a href="https://ntrs.nasa.gov/citations/19740022614">Normalised Difference Vegetation Index</a> is an application of ratioing, based on the fact that green vegetation absorbs the Red wavelength but reflects more in the NIR wavelength. This is illustrated as:</p>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="img/NDVI_spectral_trait.png" class="img-fluid figure-img" style="width:100.0%"></p>
<p></p><figcaption class="figure-caption">Fig. 3: Spectral traits of vegetation</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>Source: <a href="https://physicsopenlab.org/2017/01/30/ndvi-index/">PhysicsOpenLab</a></p>
<p><span class="math display">\[
NDVI =  \frac{NIR - Red}{NIR + Red}
\]</span></p>
<p>Applying the NDVI formula to my dataset, I obtained first</p>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="img/NDVI.png" class="img-fluid figure-img" style="width:100.0%"></p>
<p></p><figcaption class="figure-caption">Fig. 4: NDVI of Serengeti National Park</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>We can also focus on areas where NDVI is equal or greater than 0.1:</p>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="img/NDVI2.png" class="img-fluid figure-img" style="width:100.0%"></p>
<p></p><figcaption class="figure-caption">Fig. 5: NDVI of Serengeti National Park 2</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>We observe that the vegetation is not as healthy as we might expect, and this is because the images used were between 24/10/22 and 31/12/22 which coincides with the drier period.</p>
<p>This also coincides with the visualisations obtained when using the Normalised Difference Moisture Index (NDMI) and focusing on areas where NDMI is equal or greater than 0.1:</p>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="img/NDMI.png" class="img-fluid figure-img" style="width:100.0%"></p>
<p></p><figcaption class="figure-caption">Fig. 6: NDMI of Serengeti National Park</figcaption><p></p>
</figure>
</div>
</div>
</div>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="img/NDMI2.png" class="img-fluid figure-img" style="width:100.0%"></p>
<p></p><figcaption class="figure-caption">Fig. 7: NDMI of Serengeti National Park2</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p><strong>Data fusion</strong> is the process of appending new raster data to existing datasets or creating new raster datasets with different bands, and data fusion can be done with newly-created texture measures and the original data. Fusion can be done in R using the <code>stack()</code> function.</p>
<p>When merging datasets obtained from different remote sensors, all datasets should be accurately registered to one another and resampled to the same pixel size. One component-substitution method available is Principle Component Analysis (PCA). At this stage I would have done Principal Component Analysis to scale the raster datasets, but unfortunately I was unable to complete the PCA part of the practical. PCA allows comparison of data that is not easily comparable in its raw form. Performing PCA would transform the original data to produce uncorrelated principal component images (Pratt 2013). An advantage of PCA-based pan-sharpening is that the number of bands is not restricted (Klonus and Ehlers 2009)</p>
</section>
</section>
<section id="applications-of-ratio-enhancements" class="level2" data-number="3.4">
<h2 data-number="3.4" class="anchored" data-anchor-id="applications-of-ratio-enhancements"><span class="header-section-number">3.4</span> Applications of Ratio Enhancements</h2>
<p>The normalised difference vegetation index (NDVI) is widely used for vegetation studies, with the NASA Global Inventory, Monitoring and Modelling Studies (GIMMS) global coverage dataset (Tucker et al 2005) being the most widely used AVHRR (Advanced Very High Resolution Radiometer) dataset. They form a relatively robust basis for detecting long-term trends in NDVI in most of the world’s semi-arid, dry sub-humid and sub-humid areas (Fensholt and Proud 2012).</p>
<p>NDVI values can be used to determine and analyse drought-prone areas, such as Faridatul and Ahmed’s (2020) work incorporating NDVI values into a modified vegetation condition index (mVCI) which enhances the detection of agricultural drought in the study area of Bangladesh; or can be used to analyse crop yields based on Pinto et al.’s (2017) work on canola yields in Brazil.</p>
<p>Focusing on Serengeti National Park, I thought it would be useful to use NDVI on satellite imagery of SNP for potential applications of monitoring anthropogenic impacts on the Protected Area or impacts of climate change on the ecosystem. For example, Boone et al (2006) use rainfall and vegetation data (and NDVI) to model Serengeti wildebeest migratory patterns. However, Anderson et al (2010) also make the point that we should be cautious when using NDVI to study wildlife hotspots as NDVI can represent the effects of multiple, correlated processes (e.g.&nbsp;biomass, forage quality, cover for predators etc.) that influence the presence of hotspots (e.g.&nbsp;herbivore hotspots).</p>
</section>
<section id="reflections" class="level2" data-number="3.5">
<h2 data-number="3.5" class="anchored" data-anchor-id="reflections"><span class="header-section-number">3.5</span> Reflections</h2>
<p>As Andy mentioned in the lectures, it is unlikely that we would have to perform geometric corrections given that most datasets are “Analysis Ready Data” (ARD), but it is useful to know in the event that we encounter data that requires geometric correction, or even just understanding how the data products we use have been treated from their raw form.</p>
<p>Mosaicking is a useful process given the limitations of some remote sensing products regarding geographical extents, and might come in handy in future research. Nonetheless, Andy also mentioned how this process may not be as important today as it is incorporated in the Google Earth Engine (GEE) workflow, so I am excited to learn in future weeks how this process is done in GEE. Mosaicking is also a bit of a throwback personally, back to 2017-18 for me when I learnt how to orthorectify and mosaic in ERDAS Imagine software (which I realised I have forgotten most of by now) and I am excited to learn how the process has evolved today when we start using GEE.</p>
</section>
<section id="references" class="level2" data-number="3.6">
<h2 data-number="3.6" class="anchored" data-anchor-id="references"><span class="header-section-number">3.6</span> References</h2>
<p><a href="https://www.jstor.org/stable/25661201">Anderson, T. M., J. G. C. Hopcraft, S. Eby, M. Ritchie, J. B. Grace, H. Olff (2010) ‘Landscape-scale analyses suggest both nutrient and antipredator advantages to Serengeti herbivore hotspots’, <em>Ecology</em>, 91(5), 1519-29.</a></p>
<p><a href="https://www.jstor.org/stable/20069184">Boone, R. B., S. J. Thirgood, J. G. C. Hopcraft (2006) ‘Serengeti Wildebeest Migratory Patterns Modeled from Rainfall and New Vegetation Growth’, <em>Ecology</em>, 87(8), 1987-94.</a></p>
<p>Buiten, H. J. and B. Van Putten (1997) ‘Quality Assessment of Remote Sensing Registration- Analysis and Testing of Control Point Residuals’, <em>ISPRS Journal of Photogrammetry &amp; Remote Sensing</em>, 52, 57-73. Available at: <a href="https://doi-org.libproxy.ucl.ac.uk/10.1016/S0924-2716(97)83001-8" title="Persistent link using digital object identifier">https://doi.org/10.1016/S0924-2716(97)83001-8</a></p>
<p><a href="https://discovery.ucl.ac.uk/id/eprint/10113151/1/remotesensing-12-03363.pdf">Faridatul, M. I. and B. Ahmed (2020) ‘Assessing Agricultural Vulnerability to Drought in a Heterogeneous Environment: A Remote Sensing-Based Approach’, <em>Remote Sensing</em>, 12(20), 3363.</a></p>
<p><a href="https://doi.org/10.1016/j.rse.2011.12.015">Fensholt, R. and S. R. Proud (2012) ‘Evaluation of Earth Observation based global long term vegetation trends — Comparing GIMMS and MODIS global NDVI time series’, <em>Remote Sensing of Environment</em>, 119, 131-47.</a></p>
<p>Jensen, J.R. (2015) <em>Introductory Digital Image Processing,</em> 4th edn, Pearson Higher Education US (A Remote Sensing Perspective).</p>
<p><a href="https://ieeexplore-ieee-org.libproxy.ucl.ac.uk/document/5203659">Klonus, S. and M. Ehlers (2009) ‘Performance of Evaluation Methods in Image Fusion’, <em>Proceedings of the 12th International Conference on Information Fusion</em>, Seattle, July 6-9 2009, p.&nbsp;8.</a></p>
<p>Novak, K. (1992) ‘Rectification of Digital Imagery’, <em>Photogrammeric Engineering &amp; Remote Sensing</em>, 58(3), 339-44.</p>
<p><a href="https://discovery.ucl.ac.uk/id/eprint/10092180/1/1678-3921-pab-52-10-00825.pdf">Pinto, D. G., D. C. Fontana, G. A. Damalgo, E. Fochesatto, M. B. Vicari, C. Bremm, G. R. da Cunha, J. A. de Gouvea, A. Santi (2017) ‘Correlations between spectral and biophysical data obtained in canola canopy cultivated in the subtropical region of Brazil’, <em>Pesquisa Agropecuária Brasileira</em>, 52 (10), 825-32.</a></p>
<p><a href="https://learning.oreilly.com/library/view/introduction-to-digital/9781482216691/?sso_link=yes&amp;sso_link_from=university-college-london">Pratt, W. K. (2013) <em>Introduction to Digital Image Processing</em>, Boca Raton: CRC Press, p.&nbsp;736.</a></p>
<p>Tucker, C. J., J. E. Pinzon, M. E. Brown, D. A. Slayback, E. W. Pak, R. Mahoney (2005) ‘An extended AVHRR 8-km NDVI dataset compatible with MODIS and SPOT vegetation NDVI data’, <em>International Journal of Remote Sensing</em>, 26, 4485-98.</p>
<p>Wolberg, G. (1990) <em>Digital Image Warping</em>, NY: John Wiley- IEEE Computer Society, p.&nbsp;340.</p>


</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    target: function(trigger) {
      return trigger.previousElementSibling;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    setTimeout(function() {
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const cites = ref.parentNode.getAttribute('data-cites').split(' ');
    tippyHover(ref, function() {
      var popup = window.document.createElement('div');
      cites.forEach(function(cite) {
        var citeDiv = window.document.createElement('div');
        citeDiv.classList.add('hanging-indent');
        citeDiv.classList.add('csl-entry');
        var biblioDiv = window.document.getElementById('ref-' + cite);
        if (biblioDiv) {
          citeDiv.innerHTML = biblioDiv.innerHTML;
        }
        popup.appendChild(citeDiv);
      });
      return popup.innerHTML;
    });
  }
});
</script>
<nav class="page-navigation">
  <div class="nav-page nav-page-previous">
      <a href="./week2.html" class="pagination-link">
        <i class="bi bi-arrow-left-short"></i> <span class="nav-page-text"><span class="chapter-number">2</span>&nbsp; <span class="chapter-title">Week 2: Quarto and Xaringan</span></span>
      </a>          
  </div>
  <div class="nav-page nav-page-next">
      <a href="./references.html" class="pagination-link">
        <span class="nav-page-text">References</span> <i class="bi bi-arrow-right-short"></i>
      </a>
  </div>
</nav>
</div> <!-- /content -->



</body></html>